docs/en/user-guide/How-To/index.rst

```diff
@@ -3,6 +3,117 @@ How To
 
 This represents a guide on different themes about configuring in TWW.
 
+.. _settingdefaultvalues:
+
+Setting default values
+-----------------------
+
+If you get the following error message on startup of the TEKSI project during the 'Database production ready check':
+
+.. attention::
+CRITICAL: Error fk_provider or fk_dataowner not set in tww_od.default_values
+
+then your database is not yet production ready.
+
+It is possible to define default values for repeated fields such as fk_provider or fk_dataowner, so that you not have to add those manually for each object that you are creating.
+
+There is a new layer group 'configuration' in the project:
+
+.. figure:: images/configuration_default_values.png
+
+1. Open the attribute table 'Default values'
+2. Change to edit mode and choose select new object
+3. Select field name (currently fk_dataowner and fk_provider are supported in the form)
+
+.. figure:: images/default_values_fieldname_selection.png
+
+4. Select organisation that should be used (the corresponding obj_id will be added in the table itself).
+
+.. figure:: images/default_values_organisation_obj_id_selection.png
+
+
+For the second warning, please contact your TEKSI system administrator
+
+.. attention::
+CRITICAL: Error OID prefix set to 'ch000000'. Database not safe for production.
+
+and ask him/her to set the prefix to fit your project.
+
+For more information see :ref:`productionreadiness`
+
+Connect all occurences of your field name to ``tww_sys.get_default_values(field_name)``.
+Per default, the fields ``fk_provider`` and ``fk_dataowner`` are already connected to the function ``tww_sys.get_default_values(field_name)``.
+
+An example SQL script how to insert the default value for all occurences of a field name can be found `here <https://github.com/teksi/wastewater/blob/main/datamodel/changelogs/0001/14_default_values.sql>`_)
+
+
+How vw_tww_wastewater_structure labels work
+-------------------------------
+
+Labeling a manhole with his levels is quite a complex thing. Levels are not stored in the wastewater_structure class it self, but in the cover table and as reachpoint levels with the connected reaches. TWW has therefore 4 calculated fields to be able to label this levels:
```

Comment:
itself instead of it self
TWW has therefore 4 calculated fields to be able to label these levels: (es sind mehrere levels, darum these anstatt this (nur eines)